client/gutenberg/extensions/map-block/config.js

```diff
@@ -0,0 +1,338 @@
+/**
+ * Wordpress dependencies
+ */
+
+import { __ } from '@wordpress/i18n';
+
+/**
+ * External dependencies
+ */
+
+ /**
+ * Internal dependencies
+ */
+
+export const CONFIG = {
+	name: 'atavist/maps',
```

Comment:
Probably makes sense to use the same prefix like our other blocks here - `a8c`.

Reply:
Just to be sure I understand, you're suggesting replacing `atavist/` with `a8c/`? I would probably also rename the branches too, e.g. `a8c/map-block-v2`? I assume this change will break all existing instances of the block (purely testing, so no big deal) but I want to do this all at once if I'm going to make that kind of change.

Reply:
Change made in https://github.com/Automattic/wp-calypso/pull/27836/commits/be1654f671aa25df8f79a332b3dda5f2ff4c7169, going to leave the branch name intact for now.